schema/ext/vulnerability-1.0.xsd

```diff
@@ -0,0 +1,132 @@
+                    </xs:annotation>
+                    <xs:sequence>
+                        <xs:element name="url" type="xs:string" minOccurs="0" maxOccurs="1">
+                            <xs:annotation>
+                                <xs:documentation>The url of the source</xs:documentation>
+                            </xs:annotation>
+                        </xs:element>
+                    </xs:sequence>
+                    <xs:attribute name="name" type="xs:normalizedString" use="required">
+                        <xs:annotation>
+                            <xs:documentation>The name of the source</xs:documentation>
+                        </xs:annotation>
+                    </xs:attribute>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="vulnerabilityScore" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:sequence>
+                        <xs:element name="score" type="dg:scoreType" minOccurs="0" maxOccurs="unbounded"/>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="cwes" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:sequence minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="cwe" type="xs:string" minOccurs="0" maxOccurs="1"/>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="recommendations" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:annotation>
+                        <xs:documentation>The recommendations for the vulnerability</xs:documentation>
+                    </xs:annotation>
+                    <xs:sequence minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="recommendation" type="xs:string"/>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="description" type="xs:string" minOccurs="0" maxOccurs="1">
+                <xs:annotation>
+                    <xs:documentation>The description of the vulnerability</xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="advisories" minOccurs="0" maxOccurs="1">
+                <xs:complexType>
+                    <xs:annotation>
+                        <xs:documentation>The advisories for the vulnerability</xs:documentation>
+                    </xs:annotation>
+                    <xs:sequence minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="advisory" type="xs:string"/>
+                    </xs:sequence>
+                </xs:complexType>
+            </xs:element>
+        </xs:sequence>
+    </xs:complexType>
+
+    <xs:element name="vulnerabilities">
```

Comment:
Is the intended use of the vulnerabilities node to reside inside each component?

```xml
<bom>
  <components>
    <component>
      <dg:vulnerabilities>
        <dg:vulnerability>
        </dg:vulnerability>
      </dg:vulnerabilities>
    </component>
  </components>
</bom>
```

Whatever the intended location, guidance in the form of documentation would be necessary

Reply:
> Is the intended use of the vulnerabilities node to reside inside each component?

That is the intended location to be inside a component element.  Seems like a natural fit.  Would you rather have it somewhere else?

Reply:
Makes perfect sense to me as well, but the element wasn't annotated with documentation that provided any guidance. So although I assumed this to be the case, the specification did not explicitly state that.

Reply:
> guidance in the form of documentation would be necessary

Will add appropriate documentation elements that provide guidance